mpi_common.sh

```diff
@@ -43,9 +43,17 @@ function ompi_setup {
     export MPI_ARGS="-x LD_LIBRARY_PATH"
     if [ $ARCH = "x86_64" ]; then
         # Only load the OFI component in MTL so MPI will fail if it cannot
-        # be used. Also disable the Open IB BTL to avoid the call to
-        # ibv_fork_init().
-        export MPI_ARGS="$MPI_ARGS --mca pml cm --mca mtl ofi --mca btl ^openib"
+        # be used.
+        export MPI_ARGS="$MPI_ARGS --mca pml cm --mca mtl ofi"
+
+        # We have to disable the OpenIB BTL to avoid the call to ibv_fork_init
+        # EFA installer 1.10.0 (and above) ships open mpi that does not have openib btl
+        # enabled, therefore does not need the extra mca parameter
+        cur_version=$(head -n 1 /opt/amazon/efa_installed_packages | awk '{print $5}')
+        min_version=$(echo -e "$cur_version\n1.10.0" | sort --version-sort | head -n 1)
```

Comment:
I like it and I learned a new parameter to sort :)

Reply:
although it is a GNU extension, so you can't use it in a configure script.  Since we know we'll be using GNU sort here, it is fine.